### PR TITLE
Create ichimoku

### PR DESCRIPTION
**Pull request by brucetus**
Basic ichimoku cloud trading strategy. It buys when the price is above the cloud and sells when the price is inside or below the cloud.

### CODE REVIEW

File: extensions/strategies/ichimoku [hunk 1]
```diff
@@ -0,0 +1,59 @@
+var z = require('zero-fill'),
```
Comment (DeviaVir) [inline extensions/strategies/ichimoku]: I think you forgot to add the `.js` extension to this file

**Review by DeviaVir — COMMENTED** [review]
Already merged in `highest` and `lowest` (which are dependencies), so if you could only fix the filename here, this one is good to go as well :+1:

**Comment by brucetus** [comment]
Haha stupid mistake on my part. Fixed it

**Review by DeviaVir — APPROVED** [review]
:+1:

**Pull request merged**